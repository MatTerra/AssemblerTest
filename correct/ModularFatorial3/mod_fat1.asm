mod_main: begin
mod_fat: extern
n: extern
public aux
section data
aux: space
section text
input n
load n
jmp mod_fat
end
