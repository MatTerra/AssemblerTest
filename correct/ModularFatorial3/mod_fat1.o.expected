H: mod_fat1
H: 7
H: 0101010
H: U mod_fat 5
H: U n 1 3
H: D aux 6
T: 12 00 10 00 05 00 0
